src/main/java/org/mule/module/apikit/api/RamlHandler.java

```diff
@@ -112,6 +114,13 @@ public RamlHandler(ScheduledExecutorService executor,
     }
   }
 
+  private void handleWarnings(ApiReference apiReference) {
+    List<ParsingIssue> warnings = result.getWarnings();
+    if (ApiVendor.OAS_30.equals(apiReference.getVendor()) && isNotEmpty(warnings)) {
```

Comment:
Shouldn't we log all warnings ? just a question to review before merging this

Reply:
Good point! I think that doing so will be more consistent

Reply:
Updated! All warnings will be logged if any. 
Anyway, it is always possible to update the logging level of apikit by updating the the log4j2.xml of the app.
`<AsyncLogger name="org.mule.module.apikit" level="ERROR"/>`